DEVELOPING.md

```diff
@@ -48,23 +48,3 @@ an `@npm` workspace with npm dependencies.
 
 Note, with this workflow the downstream version of `@npm//typescript` will be used to compile the `ts_library` targets in `npm_bazel_typescript`.
```

Comment:
The bit about testing downstream can be removed as well. It belongs in the rules_nodejs repo and should be generalized for any @bazel/foobar package.

Reply:
okay, removing this whole file